tests/zfs-tests/tests/functional/cli_root/zpool_expand/zpool_expand_003_neg.ksh

```diff
@@ -27,23 +27,23 @@
 
```

Comment:
The additional use case and missing coverage is described in #5771 and can be addressed later.